themes/ddbasic/sass/components/node/node.scss

```diff
@@ -73,3 +73,20 @@
     }
   }
 }
+
+// Styling of cookie consent placeholders. They can be on any node type
+// and on ekursus objects.
+// The cookieinformation script automatically sets style="display: block;" on the element.
+// Therefore the need for !important override.
+.consent-placeholder {
+  display: flex !important;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #e5e5e5;
```

Comment:
As far as I can tell this is the value for [the SCSS variable `$grey`](https://github.com/ding2/ding2/blob/5c0c287e75b6d353e737b0cf48d2441d91086121/themes/ddbasic/sass/configuration/_variables.scss#L22). Is there a reason not to use it?

Reply:
Changed the css. Display block was being used to toogle the element on or off so we cant use display:flex here. Styled it in a different way. Updated background color to use variable.